pkg/generators/enum.go

```diff
@@ -74,15 +74,16 @@ func (et *enumType) ValueStrings() []string {
 // DescriptionLines returns a description of the enum in this format:
 //
 // Possible enum values:
-//  - `value1`: description 1
-//  - `value2`: description 2
+//
```

Comment:
Do we really want that empty line?

Reply:
I actually don't see it in the test, which is good.

Reply:
Sorry straw format. Fixing & squashing